util/dvsim/Deploy.py

```diff
@@ -294,6 +295,8 @@ def extract_runtimes(self, log_text: List):
                 log_text, self.sim_cfg.tool)
         except NotImplementedError:
             log.debug("Using dvsim default runtime due to unsupported tool: " + self.sim_cfg.tool)
+            self.job_runtime.time = \
```

Comment:
This may be called anytime after the job really finishes. Perhaps set the finish_time in post_finish?

Reply:
Actually this function is already implemented with Sri's PR: https://github.com/lowRISC/opentitan/pull/13952